finance-api/user-guides/fetch-account-balance.md

```diff
@@ -0,0 +1,69 @@
+Fetch account balance
+===
+Using the Finance API, you can fetch the balance that is available in the merchant's Zettle preliminary or liquid account.
+
+* [Prerequisites](#prerequisites)
+* [Step 1: Fetch the account balance](#step-1-fetch-the-account-balance)
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorization is set up using [Authorization OAuth2 API](../../authorization.adoc). 
+<!-- to be continued if any -->
+
+## Step 1: Fetch the account balance 
+Fetch the account balance in the merchant's Zettle preliminary or liquid account.  
+
+1. Optional: Fetch your organisation UUID. 
```

Comment:
Why complicate it with `users/me` when we have orgUuid in the token already and can use `self`?